pkgs/development/python-modules/beanhub-cli/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, pytz
+, pydantic
+, pydantic-core
+, pydantic-settings
+, pyyaml
+, jinja2
+, rich
+, fastapi
+, click
+, uvicorn
+, starlette-wtf
+, beancount-parser
+, beancount-black
+, beanhub-import
+, beanhub-forms
+, poetry-core
+, httpx
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "beanhub-cli";
+  version = "1.1.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "LaunchPlatform";
+    repo = "beanhub-cli";
+    rev = "refs/tags/${version}";
+    hash = "sha256-vYBbaUVJAs+aIp6aQpJb62DEDxe/sQTzgOkjPq6ADoc=";
+  };
+
+  buildInputs = [
+    poetry-core
+  ];
```

Comment:
The input names seem to have changed, if I read [the docs](https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-function) correctly this should be `build-system`.

Reply:
```suggestion
  build-system = [
    poetry-core
  ];
```